docs/mine-hnt/validators/validator-deployment-guide.mdx

```diff
@@ -0,0 +1,389 @@
+---
+id: validator-deployment-guide
+title: Test Cases
+hide_title: true
+sidebar_label: Validator Deployment Guide
+slug: /mine-hnt/validators/validator-deployment-guide
+---
+
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Validator Deployment Guide 
+
+This guide will cover the end-to-end process of deploying a Helium Validator. It assumes you're starting from scratch. At the end, you'll have a Helium Validator deployed the Network.
+
+:::info
+
+* We're currently operating the Testnet, so you'll be deploying to the Testnet until Mainnet is live. This guide reflects deploying to the Testnet. 
+* Use the #validator channel on the [Helium Discord server](https://discord.gg/helium) to take part in the real-time Validator development and get support.
+
+:::
+
+
+## Prerequisites
+
+To successfully deploy a Helium Validator, you'll need to do the following:
+
+* Install the [Helium CLI Wallet](https://github.com/helium/helium-wallet-rs)
+* Deploy a node on a cloud service like AWS, GCP, Azure, or Digital Ocean. Or deploy a physical server on which to run the validator. [The technical node requirements can be found here](/mine-hnt/validators/requirements/). 
```

Comment:
"a node" should be something else, I think. The word "node" is often used in Helium discord to talk about things running on the p2p network, so it could be confused with a running hotspot or validator or something else. I think you could say one of:
"an instance" <--- kinda AWS specific, but most folks would get it"
"a VM"
"a VPS" 
"a virtual server" <--- works well with the next sentence which has "a physical server"
"a server"